common/Clipboard.hpp

```diff
@@ -164,7 +198,24 @@ class ClipboardCache
         }
         Entry ent;
         ent._inserted = std::chrono::steady_clock::now();
-        ent._rawData = std::make_shared<std::string>(data, size);
+
+        if (size > 0x10000)
+        {
+            std::string tempFile = _cacheDir + '/' + std::to_string(_cacheFileId++);
+
+            std::ofstream fileStream;
+            fileStream.open(tempFile, std::ios::binary);
+            fileStream.write(data, size);
```

Comment:
My strong suspicion is that once we have read a huge thing into memory and allocated lots of RAM for it - we already lost quite hard.

I would really like to get the Kit process / clipboard code to back large clipboard serialization with a file stream as they are written - I'm happy to allocate memory in the Kit process since the Kernel will clean it up when it dies, and then do more of a dance to transfer that file out of the Kit folder. That then saves a load of read/write traffic on the UDS, and the big allocation and so on. I would go so far as to say we should always transfer clipboards between Kit and main process by file too to simplify things (?). More work - but more effective I think.